.gitignore

```diff
@@ -15,3 +15,4 @@ throughput
 config.toml
 config.*.toml
 .container_tmp
+.idea
```

Comment:
this probably belongs in your global gitignore

Reply:
Good point, I'll add it in my configuration.